### PR TITLE
Update SGE nodes

### PR DESCRIPTION
**Pull request by reeves89**
update coin gecko id @JeremyParish69

### CODE REVIEW

File: osmosis-1/osmosis-1.assetlist.json [hunk 1]
```diff
@@ -11242,7 +11242,7 @@
         "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/sge/images/sge.png",
         "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/sge/images/sge.svg"
       },
-      "coingecko_id": "sge",
+      "coingecko_id": "six-sigma",
```
Comment (JeremyParish69) [inline osmosis-1/osmosis-1.assetlist.json:11245]: Please make this change at the Chain Registry.

Comment (reeves89) [reply osmosis-1/osmosis-1.assetlist.json:11245]: Already did.

Comment (JeremyParish69) [reply osmosis-1/osmosis-1.assetlist.json:11245]: Oh, sorry, you indeed already had.

Comment (reeves89) [reply osmosis-1/osmosis-1.assetlist.json:11245]: No worries @JeremyParish69 .Thanks

**Pull request merged**